src/test/java/bio/terra/common/iam/AuthenticatedUserRequestTest.java

```diff
@@ -119,4 +125,108 @@ public void testVectors() throws JsonProcessingException {
             .setToken("0123.456-789AbCd")
             .build());
   }
+
+  @Test
+  public void testProtoInFlightMap() throws IOException {
+    AuthenticatedUserRequest request =
+        AuthenticatedUserRequest.builder()
+            .setEmail(EMAIL_ADDRESS)
+            .setSubjectId(SUBJECT_ID)
+            .setToken(TOKEN)
+            .build();
+
+    FlightMap inMap = request.putIn(new FlightMap(), "foo");
+    FlightMap outMap = new FlightMap();
+    outMap.fromJson(inMap.toJson());
```

Comment:
we should both update to also use the test fixture for stairway serialization added in https://github.com/DataBiosphere/stairway/pull/64 to check for input FlightMap compatibility.